app/shared/utils/StatsFetcher.js

```diff
@@ -0,0 +1,112 @@
+
+export default class StatsFetcher {
```

Comment:
Decimal.js should be included wherever these calls ultimately end up and replace all the parseFloat calls.

Reply:
right 👍